### ISSUE TITLE
Consider making `worker-timers` a parameter instead of embedding in the bundle

### ISSUE THREAD

**Issue by wbobeirne** [SupremeTechnopriest/react-idle-timer#233]
### Is your feature request related to a problem? Please describe.

The module `worker-timers` is being included by default in the bundle even though the default for using it is `false`. According to Bundlephobia, this means that `worker-timers` is currently about one third of the bundle size of this library ([`worker-timers`](https://bundlephobia.com/package/worker-timers@7.0.45) vs [`react-idle-timer`](https://bundlephobia.com/package/react-idle-timer@5.0.0-rc.18)).

### Describe the solution you'd like.

Given that it's optional, it might be nice to change to a parameter on `useIdleTimer` / `withIdleTimer` where you can pass the library as an argument instead, e.g.

```ts
import * as workerTimers from 'worker-timers';

useIdleTimer({ timers: workerTimers });
```

### Describe alternatives you've considered.

Some kind of more tree-shakeable way of doing this, like making it a `dependency` and `require`ing it. But this seems simpler to me overall given the optional nature of the workers.

**Comment by SupremeTechnopriest** [comment]
@wbobeirne 

This is an interesting suggestion.  I am definitely considering it.  May I ask why you are opting out from using the worker timers?  My original intention was to make it required since it is definitely an improvement.  I then made it optional due to CSP violations from the worker.  Regardless, I can make this more tree shakable, but that would make the default be native timers (not worker timers).

**Comment by SupremeTechnopriest** [comment]
Cool thank you for the info.  I think what I will do is make the default the main thread timers and then export workerTimers as a separate export so its tree shakable.  Then you can fulfill the `timers` interface with any set of timers you want.  I'll try to get this done today.

**Comment by SupremeTechnopriest** [comment]
Hey @wbobeirne,

This has been published as `5.0.0-rc.19`.  Please let me know if that tree shakes out ok for you.  Relevant docs can be found [here](https://idletimer.dev/docs/getting-started/new#web-worker-timers).
